### DISCUSSION TITLE
How to make a hierarchical / nested navigation rail?

### DISCUSSION THREAD

**Discussion by clstaudt** [flet-dev/flet#485] [Q&A]
I would like to arrange the items in a navigation rail into labelled groups, either by grouping them under a headline or merging them into a collapsible item.  How could this be implemented with flet? 

![Screenshot 2022-10-15 at 17 21 37](https://user-images.githubusercontent.com/875194/195994366-d8125e04-bdbb-4e9b-b2e1-dc54b7e5eeb4.png)

**Comment by FeodorFitsner** [comment]
Looks like you've already found a prototype: https://github.com/flet-dev/gallery

**Reply by clstaudt** [reply]
Great to have `Collapsible` items now. Is it also possible to group items in a navigation rail using headlines, like in the screenshot above?
